tests/globalkilltest/global_kill_test.go

```diff
@@ -111,6 +131,77 @@ func (s *TestGlobalKillSuite) connectPD() (cli *clientv3.Client, err error) {
 	return cli, nil
 }
 
+func (s *TestGlobalKillSuite) startTiKV(dataDir string) (err error) {
+	s.tikvProc = exec.Command(*tikvBinaryPath,
+		fmt.Sprintf("--pd=%s", *pdClientPath),
+		fmt.Sprintf("--data-dir=tikv-%s", dataDir),
+		"--addr=0.0.0.0:20160",
+		"--log-file=tikv.log",
+		"--advertise-addr=127.0.0.1:20160",
+	)
+	log.Infof("starting tikv: %v", s.tikvProc)
+	err = s.tikvProc.Start()
+	if err != nil {
+		return errors.Trace(err)
+	}
+	go func() {
+		err := s.tikvProc.Wait()
+		log.Info("tikv exited: ", err)
+	}()
+	time.Sleep(500 * time.Millisecond)
+	return nil
+}
+
+func (s *TestGlobalKillSuite) startPD(dataDir string) (err error) {
+	s.pdProc = exec.Command(*pdBinaryPath,
+		"--name=pd",
+		"--log-file=pd.log",
+		fmt.Sprintf("--client-urls=http://%s", *pdClientPath),
+		fmt.Sprintf("--data-dir=pd-%s", dataDir))
+	log.Infof("starting pd: %v", s.pdProc)
+	err = s.pdProc.Start()
+	if err != nil {
+		return errors.Trace(err)
+	}
+	go func() {
+		err := s.pdProc.Wait()
+		log.Info("pd exited: ", err)
+	}()
+	time.Sleep(500 * time.Millisecond)
+	return nil
+}
+
+func (s *TestGlobalKillSuite) startCluster() (err error) {
+	err = s.startPD(s.clusterId)
+	if err != nil {
+		return
+	}
+
+	err = s.startTiKV(s.clusterId)
+	if err != nil {
+		return
+	}
+	time.Sleep(10 * time.Second)
+	return
+}
+
+func (s *TestGlobalKillSuite) cleanCluster() (err error) {
```

Comment:
Can extract this function to two functions s.stopPD, s.stopTiKV?

Reply:
it is only used once, so I think it is okay not to split

Reply:
okay, I think it make sense.